plugin.audio.detektorfm/addon.xml

```diff
@@ -11,29 +11,33 @@
 		<provides>audio</provides>
 	</extension>
 	<extension point="xbmc.addon.metadata">
-		<summary lang="en_GB">detektor.fm | Internet radio with journalism and alternative pop music</summary>
+		<summary lang="en_GB">detektor.fm | German online radio combining journalism with alternative pop music</summary>
 		<summary lang="de_DE">detektor.fm | Internetradio mit Journalismus und alternativer Popmusik</summary>
-		<description lang="en_GB">German internet radio</description>
-		<description lang="de_DE">Deutsches Internetradio</description>
+		<description lang="en_GB">detektor.fm focuses on in-depth-journalism and is a platform for high-quality audio-journalism on the German speaking internet. The nationwide approach is based on a editorial code of ethics. Users can listen to content live and on-demand as the prefer.</description>
+		<description lang="de_DE">detektor.fm konzentriert sich auf die Vermittlung hintergründiger Informationen und ist somit eine Plattform für anspruchsvollen Hörfunk-Journalismus im Internet. Die national relevanten Themen werden von etablierten Journalisten nach einem eigens verfassten Kodex aufgearbeitet. So kann man im Internet ein lineares Programm anbieten und zugleich zeitsouverän Inhalte abrufen.</description>
 		<disclaimer lang="en_GB"></disclaimer>
 		<disclaimer lang="de_DE"></disclaimer>
-		<language>de</language>
+		<language>de en</language>
```

Comment:
detektor.fm has a talk- and a music-stream. the talk-stream is german only, but the music stream can be interesting for english speakers, too.
so i thought that language "de en" would be correct here ... ?